test/server/graphql/common/expenses.test.js

```diff
@@ -407,5 +408,13 @@ describe('server/graphql/common/expenses', () => {
       expect(await canUnschedulePayment(collectiveAccountantReq, expense)).to.be.false;
       expect(await canUnschedulePayment(hostAccountantReq, expense)).to.be.false;
     });
+
+    it('make sure legal name is validated against the account holder name', async () => {
+      expect(isAccountHolderNameAndLegalNameMatch('Evil Corp, Inc', 'Evil Corp, Inc.')).to.be.true;
+      expect(isAccountHolderNameAndLegalNameMatch('François', 'Francois')).to.be.true;
+      expect(isAccountHolderNameAndLegalNameMatch('Sudharaka Palamakumbura', 'Palamakumbura Sudharaka')).to.be.true;
```

Comment:
Let's add some more tests to increase the resilience here
```suggestion
      expect(isAccountHolderNameAndLegalNameMatch('Evil Corp, Inc', 'Evil Corp, Inc.')).to.be.true;
      expect(isAccountHolderNameAndLegalNameMatch('Evil Corp Inc', 'Evil Corp, Inc.')).to.be.true;
      expect(isAccountHolderNameAndLegalNameMatch(' Evil   Corp,    Inc.', '   Evil Corp   Inc')).to.be.true;
      expect(isAccountHolderNameAndLegalNameMatch('François Dêaccènt', 'Francois DeAccEnt')).to.be.true;
      expect(isAccountHolderNameAndLegalNameMatch('Sudharaka Palamakumbura', 'Palamakumbura Sudharaka')).to.be.true;
      expect(isAccountHolderNameAndLegalNameMatch('Sudharaka Palamakumbura', 'Sudharaka Palamakumbura')).to.be.true;
```

Reply:
Done. 👍🏽